doc/manual/manual.md

```diff
@@ -1542,21 +1548,73 @@ easier or more likely to succeed. For generating LLVM with `clang`, it
 can be helpful to:
 
 * Turn on debugging symbols with `-g` so that SAW can find source
-locations of functions, names of variables, etc.
+  locations of functions, names of variables, etc.
 
 * Optimize with `-O1` so that the generated bitcode more closely matches
-the C/C++ source, making the results more comprehensible.
+  the C/C++ source, making the results more comprehensible.
 
 * Use `-fno-threadsafe-statics` to prevent `clang` from emitting
-unnecessary pthread code.
+  unnecessary pthread code.
 
 * Link all relevant bitcode with `llvm-link` (including, *e.g.*, the C++
   standard library when analyzing C++ code).
 
+All SAW proofs include side conditions to rule out undefined behavior,
```

Comment:
Has anyone tested that this works reasonably well? It would be good to have a test case that is compiled with the sanitizers and demonstrates that it catches something that would otherwise be uncaught before we recommend it.

Reply:
Good point. One of the things I'm focused on this week is writing some more tests, and this would be a good one.

Reply:
One specific case I can recall: clang is happy to compile a signed right shift to a shift that doesn't have the `nsw` flag.  Turning on the `-fsantitize=shift` option correctly found that overflow condition.

Reply:
I'm glad you brought this up, @langston-barrett, because it actually doesn't work as-is. Using `-fsanitize=blah` introduces a call to a special function for each type of failure, and we don't have overrides for any of them, so you get a confusing error (that a pointer is not a function pointer). I think I'll hold off on recommending the use of the sanitizer for now.